tensorflow/core/common_runtime/bfc_allocator.h

```diff
@@ -378,7 +378,7 @@ class BFCAllocator : public VisitableAllocator {
   inline int Log2FloorNonZero(uint64 n) {
 #if defined(__GNUC__)
     return 63 ^ __builtin_clzll(n);
-#elif defined(PLATFORM_WINDOWS)
+#elif defined(PLATFORM_WINDOWS) && (_WIN64)
```

Comment:
Just change this line to `#elif defined(_WIN64)` since it only applies to Windows 64-bit.